types/node/net.d.ts

```diff
@@ -621,7 +621,7 @@ declare module "net" {
          * @since v0.1.90
          * @param callback Called when the server is closed.
          */
-        close(callback?: (err?: Error) => void): this;
+        close(callback?: (err?: Error | null) => void): this;
```

Comment:
I don't believe this to be the case? The referenced change was specific to `Writable.prototype.write`, which doesn't affect this method. Looking at the source and from trivial testing, this callback appears to be called with no arguments when the socket is successfully closed?

Reply:
On closer look, yes, I think you're right -- https://github.com/nodejs/node/issues/47261 _requested_ a change, it hadn't yet made it.  My bad, sorry for the miss on this one.